### PR TITLE
[FLORA-450] Add a guide for namespaces

### PR DESCRIPTION
**Pull request by tchoutri**
## Proposed changes

image

## Contributor checklist

- [x] My PR is related to #450
- [x] I have read and understood the [CONTRIBUTING guide](https://github.com/flora-pm/flora-server/blob/development/CONTRIBUTING.md)
- [ ] I have inserted my change and a link to this PR in the [CHANGELOG](https://github.com/flora-pm/flora-server/blob/development/CHANGELOG.md)

### CODE REVIEW

File: docs/docs/namespaces.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    443abb7fb497a134c343faf52f0b659bd7999bc06b7f63fa76dc99d631f9bea1
+    a86a1f6ce86c449c46666bda44268677abf29b5b2d2eb5ec7af903ec2f117a82
+    bcec67e8e99cabfa7764d75ad9b158d72bfacf70ca1d0ec8bc6b4406d1bf8413
+    c00aae8461a256275598500ea0e187588c35a5d5d7454fb57eac18d9edb86a56
+    d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
+```
+and run `cabal update`.
+
+[`@haskell`]: https://flora.pm/packages/@haskell
+[`@cardano`]: https://flora.pm/packages/@haskell
```
Comment (RaoulHC) [inline docs/docs/namespaces.md:39]:
```suggestion
[`@cardano`]: https://flora.pm/packages/@cardano
```

File: docs/docs/namespaces.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Some packages are foundational to the ecosystem and maintained by either the Core Libraries Committee or the GHC team, and this makes them unique in terms of the expectations we have from them.
+
+These packages live in the [`@haskell`] namespace to show that they are stable and reliable. Some example of packages are `base`, `text`, `bytestring`, and `mtl`.
+
+## @hackage Packages
+
+This is where most third-party packages live, which you will find on [Hackage](https://hackage.haskell.org).
+
+## @cardano Packages
```
Comment (RaoulHC) [inline docs/docs/namespaces.md:18]: Is it worth putting a brief word on (or a link to) the Cardano index? I've heard of CHaP but had to look up what its purpose was.

**Review by RaoulHC — COMMENTED** [review]
Two small points, but looks good. I look forward to when there's a flora namespace!

File: docs/docs/namespaces.md (hunk 2 above)
Comment (tchoutri) [reply docs/docs/namespaces.md:18]: The link in the first sentence doesn't work for you?

File: docs/docs/namespaces.md (hunk 1 above)
Comment (tchoutri) [reply docs/docs/namespaces.md]: Thanks!

File: docs/docs/namespaces.md (hunk 2 above)
Comment (RaoulHC) [reply docs/docs/namespaces.md:18]: Doh, hadn't rendered and didn't realise you could do links via footnotes in markdown.

Comment (tchoutri) [reply docs/docs/namespaces.md:18]: oh yeah it's very useful. :)

**Review by RaoulHC — APPROVED** [review]
LGTM

**Pull request merged**